src/core/action-creator.ts

```diff
@@ -50,14 +50,14 @@ export function createActions(flux: FluxCapacitor) {
 
       addRefinement: (field: string, valueOrLow: any, high: any = null) => {
         if (typeof high === 'number') {
-          actions.updateSearch({
+          return actions.updateSearch({
```

Comment:
we should add tests to show that we expect this to return so we don't have this problem again